packetbeat/etc/packetbeat.template.json

```diff
@@ -27,6 +27,24 @@
         "client_location": {
           "type": "geo_point"
         },
+        "dns": {
+          "properties": {}
+        },
+        "http": {
+          "properties": {}
+        },
+        "icmp": {
+          "properties": {}
+        },
+        "memcache": {
+          "properties": {}
+        },
+        "mongodb": {
+          "properties": {}
+        },
+        "mysql": {
+          "properties": {}
```

Comment:
No, it just happened based on the changes. Not sure if it has any side affects. Probably to keep the templates short (and prevent side affects) we should check for empty properties and not write them?